charts/kube-prometheus-stack/templates/alertmanager/alertmanager.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
   labels:
     app: {{ template "kube-prometheus-stack.name" . }}-alertmanager
 {{ include "kube-prometheus-stack.labels" . | indent 4 }}
+{{- if .Values.alertmanager.additionalLabels }}
+{{ toYaml .Values.alertmanager.additionalLabels | indent 4 }}
+{{- end }}
```

Comment:
```suggestion
    {{- include "kube-prometheus-stack.labels" . | nindent 4 }}
    {{- with .Values.alertmanager.additionalLabels }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
```

Reply:
I can do the change, no problem. But is there a reason to do it differently than how the annotations are added two lines below?

Reply:
Thats the leagcy style as well, but I can't add suggestions there. At least each time - legacy code it touched, I suggest to indent them. 

Feel free to do that for the annotations as well.